development/build/utils.js

```diff
@@ -0,0 +1,21 @@
+function getNextBetaVersionMap(currentVersion, platforms) {
+  // eslint-disable-next-line prefer-const
+  let [major, minor, patch, premajor] = currentVersion.split('.');
+  minor = Number(minor) + 1;
+  patch = 0;
+
+  return platforms.reduce((platformMap, platform) => {
+    if (platform === 'firefox') {
+      premajor = 'beta0';
+    } else {
+      premajor = '0';
```

Comment:
For this we should create a command line param or environment variable where we allow the beta premajor to be passed in, in the rare event we have multiple beta versions.